pkg/configuration/configuration.go

```diff
@@ -129,6 +132,7 @@ func (c *Config) setConfigDefaults() {
 	c.host.SetDefault(VarDurationBeforeChangeRequestDeletion, "24h")
 	c.host.SetDefault(varNotificationDeliveryService, NotificationDeliveryServiceMailgun)
 	c.host.SetDefault(varDurationBeforeNotificationDeletion, "24h")
+	c.host.SetDefault(varRegistrationServiceURL, "https://registration.crt-placeholder.com")
```

Comment:
Reg service URL will depend on the particular cluster/env. Not sure how it's best to setup it...
@tinakurian I guess we can add a config map name to the environments as we did for secrets: https://github.com/codeready-toolchain/host-operator/blob/master/deploy/env/prod.yaml#L17
Something like this:
```
host-operator:
  config-map:
    name: host-operator-config
```
And load all keys/values from it when we loading the configuration. Similar to the secret loading.
Then we can create a config map with reg service URL in particular clusters.

Reply:
sounds good, i can add this!

Reply:
done here: https://github.com/codeready-toolchain/toolchain-infra/pull/21 and https://github.com/codeready-toolchain/host-operator/pull/244